packages/core/src/browser/tree/tree-search.ts

```diff
@@ -73,15 +65,10 @@ export class FileNavigatorSearch implements Disposable, TreeDecorator {
             transform
         });
         this._filteredNodes = this._filterResult.map(match => match.item);
-        this.fireDidChangeDecorations(() => new Map(this._filterResult.map(m => [m.item.id, this.toDecorator(m)] as [string, TreeDecoration.Data])));
         this.fireFilteredNodesChanged(this._filteredNodes);
         return this._filteredNodes!.slice();
     }
 
-    get onDidChangeDecorations(): Event<(tree: Tree) => Map<string, TreeDecoration.Data>> {
```

Comment:
TODO: one has to verify any possible resource leaks. (https://github.com/theia-ide/theia/pull/2063#issuecomment-395290511)

Reply:
Checked, didn't reproduce.